crytic_compile/platform/vyper.py

```diff
@@ -62,8 +62,10 @@ def compile(self, crytic_compile: "CryticCompile", **kwargs: str):
         crytic_compile.abis[contract_name] = info["abi"]
         crytic_compile.bytecodes_init[contract_name] = info["bytecode"].replace("0x", "")
         crytic_compile.bytecodes_runtime[contract_name] = info["bytecode_runtime"].replace("0x", "")
-        crytic_compile.srcmaps_init[contract_name] = []
-        crytic_compile.srcmaps_runtime[contract_name] = []
+        crytic_compile.srcmaps_init[contract_name] = [info["source_map"]["pc_pos_map_compressed"]]
```

Comment:
`pc_pos_map_compressed` is the same for init and runtime bytecode, I am assuming this is not correct?

Reply:
I think it is the case, but I'm not 100% sure about it.